sig-contributor-experience/charter.md

```diff
@@ -71,26 +65,42 @@ We cross-cut all SIGs and WGs to deliver the following processes:
 If we need funding for any reason, we approach [CNCF].
 CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initiatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
 
+- External communications platforms that we do not control but might have some influence over
+    - [r/kubernetes]
+    - [@kubernetesio] twitter account
+    - [Kubernetes Blog]
+
 ### Out of scope
 
 - Code for the testing and CI infrastructure - that’s SIG Testing
-- [kubernetes/community]  ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
-- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
-- The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Management] subproject README.
-- Steering committee election policy updates and maintenance.
+- [kubernetes/community] Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
```

Comment:
what is those folders?

Reply:
Design proposals technically still live in k/community. They haven't been migrated to k/enhancements yet, but that is the intended home AFAIK.

Reply:
The intention of this is describing the fact that we own the repo as a whole, and the operational processes around the repo, but we delegate certain subfolders to other sigs/parts of the project. For example, we own the sig docs generator and the root, but we don't own the steering committee folder, or the the various sig folders. If we delegate it, we don't own it.